### PR TITLE
update milet configuration

### PR DESCRIPTION
**Pull request by lsteinmann**
Hey guys, I have a final update before our excavations starts this year. (If you plan on releasing a patch again, great - if not, no problem for us, as we have a running version at least for windows.) Just some info: I was unable to try it with your latest commit, sadly. This configuration runs without any problems with Patch 2, though. Also, I do not know if you want to integrate this: we added a french translation for basically everything, so there are now -fr files in in /Core/ and /Library/. If you have any questions or if I need to change anything, please let me know.

contains
* French Translations of everything
* Small Changes to Pottery, Sample,  and Building Forms
* Switched input types on some forms
* Updated Valuelists (custom period valuelist entered everywhere, custom stone material list)
Last update before Excavation 2021

### CODE REVIEW

**Comment by lsteinmann** [comment]
Okay, it was not working, because of the Language-Milet...-files. I had renamed commons and general fields, and your new setup does not allow that. So, I decided (to make it simple) to delete the offending lines for now, though I would like those translations. Maybe you could make it possible some time in the future? Second problem is: I removed one valuelist ("Pottery-analysisMethod-milet") before due to restructuring, and now no matter what I try to do, I cannot use the configuration without this valuelist in the Valuelists-file. Funny enough, that is only true for my "usual" project, which had test data using that particular valuelist, while an empty project with the same configuration worked fine. Now, I deleted the test data in question, but nothing changes. In the end, I restored the valuelist (somewhat empty), and it works - though this is not pretty, I guess, and somewhat weird. 

Long story short: Latest commit works fine with your current commit. 

Anyway, hope you have a good weekend!

**Comment by tkleinke** [comment]
Hi, sorry for your problems with the master branch. We're currently adding a configuration editor to the application and a lot of things are still work in progress. What's happing now is the following: The first time a project is loaded, its configuration file is stored in the database and from then on only this copy in the database is used (to allow editing it via the new configuration editing tools). So your changes to the configuration files are only applied when creating a new project or switching to one you haven't opened with the master branch version yet. You can also force reloading the file by manually removing the document with id "configuration" from the database. Anyway, I'll remove the valuelist after merging the pull request.

Renaming common fields is allowed, but currently only for specific categories (you can set a label/description there like you can for any other field). We'll probably allow changing translations for all identifier fields etc. at once at a later time, but that's not possible now with the master branch version. I'll add the labels & descriptions from your original commit to the patch branch though, so they'll be delivered with the next patch coming later this week.

And last but not least: many thanks for the French translations! :)
